sdk/servicebus/test-resources.json

```diff
@@ -99,6 +111,10 @@
     "SERVICEBUS_CONNECTION_STRING": {
       "type": "string",
       "value": "[listkeys(variables('authRuleResourceId'), variables('sbVersion')).primaryConnectionString]"
+    },
+    "SERVICEBUS_SECONDARY_NAMESPACE_CONNECTION_STRING": {
```

Comment:
Curious.   Why do we need two namespaces?

Reply:
The SB function extension tests require it.